practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>"These will be replaced within the model by learnable embedding features." -&gt; "These will be replaced within the model by learnable embedding features (i.e. every user will have an embedding which is learned during the training of the model)."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Resolved in next commit thanks :) </p>